appengine/js/js-blocks.js

```diff
@@ -159,11 +161,13 @@ Blockly.Blocks['logic_compare'].init = function() {
 Blockly.Msg.LOGIC_OPERATION_AND = '&&';
 Blockly.Msg.LOGIC_OPERATION_OR = '||';
 
+Blockly.Msg.LOGIC_NEGATE_TITLE = '! %1';
```

Comment:
Sadly that's overriding an existing message name.  I named this badly 5 years ago (back when blocks had titles).  Can't change that without coordinating with Translatewiki to rename it in 50 places, and breaking lots of 3rd party projects.